content/authors/escott/_index.md

```diff
@@ -8,43 +8,44 @@ social:
 #   link: /#contact
 - display:
     header: true
-  icon: twitter
+- icon: twitter
   icon_pack: fab
-  link: <<twitter page>>
-# - icon: google
-#   icon_pack: fab
-#   link: <<google scholar page>>
-# - icon: orcid
-#   icon_pack: fab
-#   link: <<orcid page>>
-# - icon: github
-#   icon_pack: fab
-#   link: <<github page>>
+  link: https://twitter.com/LeafyEricScott
+- icon: google
+  icon_pack: fab
+  link: https://scholar.google.com/citations?user=kPEDEO0AAAAJ&hl=en
+- icon: orcid
+  icon_pack: fab
+  link: https://orcid.org/0000-0002-7430-7879
+- icon: github
+  icon_pack: fab
+  link: https://github.com/Aariq
 
-# bio: 
-# education:
-#   courses:
-#   - course: Ph.D. in <<field>>
-#     institution: <<institution>>
-#     year: <<year>>
-#   - course: M.S. in <<field>>
-#     institution: ----
-#     year: <<year>>
-#   - course: B.S. <<field>>
-#     institution: <<institution>>
-#     year: <<year>>
+bio: Eric is a postdoctoral researcher in the Bruna Lab working on the effects of drought and habitat fragmentation on *Heliconia acuminata* population dynamics.
+education:
+  courses:
+  - course: PhD in Biology
+    institution: Tufts University
+    year: 2020
+  - course: MS in Ecology, Evolution, and Conservation Biology
+    institution: University of Illinois at Urbana-Champaign
+    year: 2010
+  - course: BA in Biology
+    institution: Whitman College
+    year: 2006
 email: "scottericr@gmail.com"
 # highlight_name: true
-# interests:
-# - <<keyword>>
-# - <<keyword>>
-# - <<keyword>>
-# - <<keyword>>
+interests:
+- Chemical ecology
+- Eco-metabolomics
+- Plant demography
+- Statistics and programming in R
+- Tea
+    
 organizations:
-- name: Department of Wildlife Ecology and Conservation
-- name: University of Florida
-  url: <<your website>>
+- name: University of Florida, Wildlife Ecology & Conservation
+  url: "https://wec.ifas.ufl.edu/"
```

Comment:
Pretty sure this is supposed to be the URL for the organization, not my personal website, but feel free to change it after seeing what it looks like when built.

Reply:
Ah, I see what you mean - yes, I wanted "personal website", burt in that location it is for the institutional one. Thx.